Add documentation to refusal, cancellation, change seller in fulfillment (#36)
vtex/checkout-instore-custom: physical-fulfillment/README.md

# Physical Fulfillment configurations at checkout-instore-custom

### fulfillment-feature-config-example

This feature enable/disable fulfillment on the store. The features in fulfillment are:
1. shipFromStore: Enable ship from store order fulfillment
2. pickupInStore: Enable pickup in store order fulfillment
3. partialPicking: Enable partial picking (the default configuration is enabled)
4. reverseOrder: Enable order reversal (the default configuration is disabled)
5. newFulfillmentList: Display the fulfillment list BETA (This config is temporary) (Default: Disabled)

### Cancellation / Refusal / Change Seller configurations

> There are 2 configuration flags _(+1 deprecated)_ that allows customize cancellation order flow in Fulfillment:

Reviewer: ```suggestion
There are 2 configuration flags _(+1 deprecated)_ that allow customizing cancellation order flow in Fulfillment:
```

> - *refusalAnalysis (default: false):* operator will refuse order and sent it to analysis by an admin user that will take some action later. So the order will not be cancelled and status will be set to `refusal_analysis`.

Reviewer: ```suggestion
- *refusalAnalysis (default: false):* operator will refuse order and send it to an analysis by an admin user that will take some action later. So the order will not be cancelled and status will be set to `refusal_analysis`.
```

> - If `refusalAnalysis` configuration is not enabled, the order will display a cancel action.

Reviewer: ```suggestion
- If `refusalAnalysis` configuration is not enabled, the system will display a cancel action button, instead of refuse.
```

- *automaticChangeSeller (default: true):*  Trigger a change seller action and if there is a new seller available with equal or better price and SLA delivery date, the order will be automatically changed to it. If there is not any available seller with equal or better conditions the flow will follow doing the next action: cancellation or refusal.

> - *allowStoresToCancelOrder (_deprecated_):* Changes the "refuse order" button for "cancel order" enabling to operator cancel order directly. This flag will work only if none of the two other configurations are not set. This configuration is deprecated and will be removed soon, so to enable/disable refusal/cancelation use `refusalAnalysis` configuration as explained above.

Reviewer: ```suggestion
- *allowStoresToCancelOrder (_deprecated_):* Changes the "refuse order" button for "cancel order" enabling the operator to cancel the order directly through the system. This flag will work only if none of the two other configurations are set. This configuration is deprecated and will be removed soon, so to enable/disable refusal/cancelation use `refusalAnalysis` configuration as explained above.
```

With these two main configurations above it's possible to create 4 possible scenarios [exemplified here](./fulfillment-on-cancel-refuse-change-seller-example.js):
1. Cancel order directly.

> 2. Automatic Change Seller + Cancel order (DEFAULT SCENARIO): try to change seller a and if has not seller available, cancel ther order.

Reviewer: ```suggestion
2. Automatic Change Seller + Cancel order (DEFAULT SCENARIO): try to change seller and if has no seller available, cancel the order.
```

3. Refuse order to analysis directly.

> 4. Refuse order to analysis + Automatic Change Seller: try to change seller a and if has not seller available, refuse order and send it to analysis.

Reviewer: ```suggestion
4. Refuse order to analysis + Automatic Change Seller: try to change seller and if has no seller available, refuse the order and send it to analysis.
```
